exaparser/workflow/units/execution/modeling/aiida.py

```diff
@@ -0,0 +1,37 @@
+from express import ExPrESS
+
+from . import ModelingExecutionUnit
+
+
+class AiidaExecutionUnit(ModelingExecutionUnit):
+    """
+    Structures from AiiDA archive parser class.
+
+    Args:
+        config (dict): structures config.
+        work_dir (str): full path to working directory.
+    """
+
+    def __init__(self, config, work_dir):
+        self.config = config
+        self.work_dir = work_dir
+
+    def structures(self):
+        express = ExPrESS('aiida-archive', workDir=self.work_dir)
+        return express.parser.structures()
+
+    def to_json(self):
+        """
+        Returns the structures in JSON format.
+
+        Returns:
+             dict
+        """
+        return {
+            "structures": self.structures(),
+            "isMultiMaterial": True  # to let job have multiple materials
+        }
+
+    type = 'execution'
+    status = ''
+    application = 'aiida'
```

Comment:
TODO: This needs to be implemented properly on the next iteration.

Reply:
Also it looks like the other ModelingExecutionUnit children return a dict for application, should this not do the same?

Reply:
Fixed.